src/initIcons.ts

```diff
@@ -0,0 +1,4 @@
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { faTimes } from '@fortawesome/free-solid-svg-icons'
+
+library.add(faTimes)
```

Comment:
@abbyoung fyi I started this file which lets us use the Global pattern documented [here](https://fontawesome.com/v5.15/how-to-use/on-the-web/using-with/react), it should make adding more icons a little easier! I also had to import to all entrypoints (app, storybook, jest).